build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
     id 'org.owasp.dependencycheck' version '3.2.1'
     id 'com.github.ben-manes.versions' version '0.17.0'
     id 'net.ltgt.apt' version '0.14'
-    id "org.flywaydb.flyway" version "5.0.7"
+    id "org.flywaydb.flyway" version "5.2.3"
```

Comment:
Dependancy bot has changes version of flyway from 5.2.1 to 5.2.3. I think in order to fix the build we should downgrade from 5.2.3 to 5.2.1.  The change of line was compile group: 'org.flywaydb', name: 'flyway-core', version: '5.2.3'

Reply:
Have tried that too and it didn't work. Wanted to see if upgrading the lougin helped. Without the real error we are shooting in the dark.